lib/tizen_plugins.dart

```diff
@@ -629,3 +633,82 @@ Future<void> renderTemplateToFile(
   await file.create(recursive: true);
   await file.writeAsString(renderedTemplate);
 }
+
+/// See: [_writeFlutterPluginsList] in flutter_plugins.dart
+Future<void> _writeTizenPluginsExtraInfo(
+  FlutterProject project,
+) async {
+  YamlMap? packagesFromPubspecLock() {
+    final File pubspec = project.directory.childFile('pubspec.lock');
+    if (!pubspec.existsSync()) {
+      return null;
+    }
+
+    Object? contents;
+    try {
+      contents = loadYaml(pubspec.readAsStringSync());
+    } on YamlException catch (err) {
+      globals.printTrace('Failed to parse packages from pubspec.lock: $err');
+    }
+    if (contents == null || contents is! YamlMap) {
+      return null;
+    }
+    final Object? packages = contents['packages'];
+    if (packages == null || packages is! YamlMap) {
+      return null;
+    }
+    return packages;
+  }
+
+  final File appDepsJson = project.directory.childFile('app.deps.json');
+
+  final List<TizenPlugin> plugins = await findTizenPlugins(project);
+  final List<Map<String, Object>> pluginInfo = <Map<String, Object>>[];
+  final YamlMap? packages = packagesFromPubspecLock();
+  for (final TizenPlugin plugin in plugins) {
+    String version = '';
+    if (packages != null && packages.containsKey(plugin.name)) {
+      final YamlMap package = packages[plugin.name] as YamlMap;
+      if (package.containsKey('version')) {
+        version = package['version'] as String;
+      }
+    }
+    pluginInfo.add(<String, Object>{
+      'name': plugin.name,
+      'version': version,
+    });
+  }
+
+  final Map<String, Object?> result = <String, Object>{};
+  result['info'] =
+      'This is a generated file; do not edit or check into version control.';
```

Comment:
It looks like 'info' isn't needed.

Reply:
This file is created at packaging time and is created in the build directory. Therefore, there is no situation when the version-control program can detect it. so, I think it is unnecessary.

Reply:
Flutter makes plugins information files when running flutter commands.
When the Flutter-tizen collects plugins information, this file also is created, and located in flutter project by default. 
And then, it is copied to res folder during packaging time. 
So, I think it need to be remained.

Reply:
um.. Apart from deleting the info text, I think the problem is that the app.deps.json file still remains in the working directory.
```
$ find -name "app.deps.json"
./app.deps.json
./build/tizen/tpk/tpkroot/res/flutter_assets/app.deps.json
./tizen/flutter/ephemeral/res/flutter_assets/app.deps.json
./tizen/bin/Release/tizen80/tpkroot/res/flutter_assets/app.deps.json
```
This file should only be included in the package file at packaging time, so there is no need to keep it in flutter_assets. And the file should also be deleted from the working root directory.

`./app.deps.json`
`./tizen/flutter/ephemeral/res/flutter_assets/app.deps.json`

Reply:
As we discussed offline, I have applied them.
- Change the file name from app.deps.json to .app.deps.json
- Move it to app_root/tizen/
- Add its name to app_root/tizen/.gitignore